main.go

```diff
@@ -348,6 +381,14 @@ func mainCore() error {
 		// Wait for the results
 		return waitForSync(sqliteSyncRes, pgSyncRes, usePG, quit)
 	}
+
+	// Allow Ctrl-C to halt startup
```

Comment:
Nothing remotely time-consuming happens between https://github.com/decred/dcrdata/pull/699/files#diff-7ddfb3e035b42cd70649cc33393fe32cR302 and here.  I suggest just keeping this one.

Actually, maybe move this one down to right before `explore.StartMempoolMonitor` since Collect() and Store() could take some time during which ctrl+c could be hit, but those the only significant operations that are not already listening on `quit` between https://github.com/decred/dcrdata/pull/699/files#diff-7ddfb3e035b42cd70649cc33393fe32cR223 and down there before `explore.StartMempoolMonitor`